### PR TITLE
Let's see how this goes.

### PR DESCRIPTION
**Pull request by mingwandroid**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mingwandroid** [comment]
Can anyone help figure this out?

```
  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 43, in __call__
    return super(MatchSpecType, cls).__call__(**parsed)
  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 180, in __init__
    self._match_components = self._build_components(**kwargs)
  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 367, in _build_components
    return frozendict(_make(key, value) for key, value in iteritems(kwargs))
  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 367, in <genexpr>
    return frozendict(_make(key, value) for key, value in iteritems(kwargs))
  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 361, in _make
    matcher = _implementors[field_name](value)
  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda/models/version.py", line 439, in __new__
    raise InvalidVersionSpecError(spec)
conda.exceptions.InvalidVersionSpecError: Invalid version spec: $
```

I cannot reproduce it locally using the following command line (and also saw no weird specs with conda-render):

```
conda-build -m ./.ci_support/osx_.yaml -c conda-forge recipe
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,129 +44,147 @@ build:
...
+    - {{native}}tk                       # [win]
+    - {{native}}zlib 1.2.8               # [win]
+    - {{native}}gsl                      # [win]
+    - {{native}}libtiff >=4.0.3,<4.0.8   # [win]
+    - {{native}}libxml2 2.9.*            # [win]
+    - {{native}}openblas                 # [win]
+    # Once run_exports are in place these will not be necessary:
+    - {{ pin_compatible('pango') }}      # [not win]
+    - {{ pin_compatible('bzip2') }}      # [not win]
+    - {{ pin_compatible('icu') }}        $ [not win]
```
Comment (bgruening) [inline recipe/meta.yaml]: @mingwandroid this is the typo. Also there is a meta.yaml.orig included in this PR.

Comment (mingwandroid) [reply recipe/meta.yaml]: Urgh! How did I not spot that (and why is it not happening locally .. questions for myself)? OK. thanks.

**Comment by mingwandroid** [comment]
@conda-forge/r please review.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -38,129 +44,146 @@ build:
...
+  # are overly restrictive IMHO, should be
+  # open-ended, but actually because we
+  # need to copy the shared library into this
+  # package and change its SONAME to libRblas.so
+  # we do not want a run dependency on openblas
+  # at all (otherwise Matrix.so ends up with a
+  # DT_NEEDED of libopenblas.so and cannot be used
+  # against any old libRblas.so)
+  ignore_run_exports:
+    - libopenblas
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: Should this be `{{native}}openblas`

Comment (mariusvniekerk) [inline recipe/meta.yaml]:
```suggestion
    - {{native}}openblas
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,21 +10,27 @@ source:
...
+    - 0014-Add-luuid-to-X_PRE_LIBS.patch
+    - 0015-link-Xt-to-uuid.patch
+    - 0016-Mimic-OpenBLAS-when-making-reference-Rblas-and-Rlapa.patch
+    - 0017-Foribly-remove-then-forcibly-mv-in-javareconf.in.patch
 
 build:
-  number: 1
-  merge_build_host: True   # [win]
+  merge_build_host: True  # [win]
+  number: 4
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]:
```suggestion
  number: 1004
```

We need this for the dual compiler parts

File: recipe/activate-r-base.sh [hunk 4]
```diff
@@ -1,8 +1,2 @@
 #!/usr/bin/env sh
 R CMD javareconf > /dev/null 2>&1 || true
-
-# store existing RSTUDIO_WHICH_R
```
Comment (mariusvniekerk) [inline recipe/activate-r-base.sh]: Is this safe to remove?

File: recipe/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/meta.yaml]: No, the m2w64-* packages don't have run_exports in them. Therefore they need to be pinned.

**Comment by mingwandroid** [comment]
@isuruf, msys2 packages use an epoch package / mechanism to manage pinning. It's different than our usual way but works just fine! I'd rather not pin beyond that personally since then you have two mechanisms in play.

**Comment by mingwandroid** [comment]
Given the two points yes. Depend on the exact epoch version here, good point. I may rebuild them later for exact pinning too.

**Comment by mingwandroid** [comment]
Thanks for taking this further.

**Comment by mariusvniekerk** [comment]
Restarted travis which seemed to die for no good reason.

**Comment by mariusvniekerk** [comment]
closes #60

**Comment by scopatz** [comment]
Thanks @mingwandroid!

**Comment by CJ-Wright** [comment]
Woooooo awesome! Thank you everyone for your hard work and perseverance!!

File: recipe/yum_requirements.txt.bak [hunk 5]
```diff
@@ -1,8 +0,0 @@
-# TODO get rid of this and use xorg package.
-#mesa-libGL-devel
```
Comment (jakirkham) [inline recipe/yum_requirements.txt.bak]: Note: Looks like this was left over from a merge conflict tool. Maybe it should be dropped?

**Pull request merged**